pkg/controller/volume/expand/cache/volume_resize_map.go

```diff
@@ -272,7 +303,23 @@ func (resizeMap *volumeResizeMap) addBackResizeRequest(pvcr *PvcWithResizeReques
 		glog.V(5).Infof("Found another resize request pending for volume %s", pvcr.QualifiedName())
 		return
 	}
-	resizeMap.pvcrs[pvcUniqueName] = pvcr
+	pvc, err := resizeMap.kubeClient.CoreV1().PersistentVolumeClaims(pvcr.PVC.Namespace).Get(pvcr.PVC.Name, metav1.GetOptions{})
```

Comment:
Use shared informers please.

Reply:
This object doesn't have informers. It can be supplied to it, if need be - but I am not sure if it should be a blocker. Doing one off, GET should be fine directly via kubeClient?